### PR TITLE
cmd: add dvc update --to-remote docs

### PR DESCRIPTION
**Pull request by isidentical**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/update.md [hunk 1]
```diff
@@ -49,6 +49,17 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
+- `--to-remote` - sync the changes directly on the remote storage, rather than
```
Comment (shcheklein) [inline content/docs/command-reference/update.md:52]: 1. it's not very descriptive. Can we at least reference `--to-remote` for the `dvc add`/`dvc import-url`. That this option corresponds to those, etc.
2. does it work for the regular `dvc import`? what happens if I run it?

**Review by shcheklein — CHANGES_REQUESTED** [review]
Trying to understand the semantics first, to see how to write it down properly.

File: content/docs/command-reference/update.md [hunk 2]
```diff
@@ -49,6 +49,17 @@ $ dvc update --rev master
...
+  on the local <abbr>workspace</abbr>.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/update.md:53]: Agree on Ivan's Q. For now (based on the published text in https://dvc.org/doc/command-reference/import-url#options here's a first copy edit (not necessarily final but we could commit it for now):

```suggestion
- `--to-remote` - update the external data, but don't move it into the
  workspace, nor cache it.
  [Transfer it](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  directly to remote storage (the default one, unless `-r` is specified)
  instead. Use `dvc pull` to get the data locally.
```

File: content/docs/command-reference/update.md (hunk 1 above)
Comment (isidentical) [reply content/docs/command-reference/update.md:52]: > does it work for the regular dvc import? what happens if I run it?

iterative/dvc#5475 should forbid using with import (currently you get an irrelevant error).

Comment (isidentical) [reply content/docs/command-reference/update.md:52]: > it's not very descriptive. Can we at least reference --to-remote for the dvc add/dvc import-url. That this option corresponds to those, etc.

@jorgeorpinel 's suggestion seems fixing it, I can commit it if there is nothing blocks?

File: content/docs/command-reference/update.md (hunk 2 above)
Comment (shcheklein) [reply content/docs/command-reference/update.md:53]: `update the external data` sounds like we are updating the data itself. Can we say something like "update .dvc files and move new version of the external data into the remote storage ..."?

File: content/docs/command-reference/update.md (hunk 1 above)
Comment (shcheklein) [reply content/docs/command-reference/update.md:52]: resolving this one, let's keep discussion in the @jorgeorpinel suggestion ...

File: content/docs/command-reference/update.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/update.md:53]: Sure, maybe this then:

```suggestion
- `--to-remote` - update the import `.dvc` file and
  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  the new data directly to remote storage (the default one unless `-r` is used).
  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
  data locally.
```

**Comment by isidentical** [comment]
Hey @shcheklein @jorgeorpinel are we missing anything on this?

**Review by jorgeorpinel — APPROVED** [review]

File: content/docs/command-reference/update.md [hunk 3]
```diff
@@ -49,6 +49,20 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
+- `--to-remote` - update the import `.dvc` file and
+  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
+  the new data directly to remote storage (the default one unless `-r` is used).
+  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
+  data locally.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/update.md:56]: We forgot to mention that this only works for `import-url` imports @isidentical — per iterative/dvc/pull/5475

Comment (jorgeorpinel) [reply content/docs/command-reference/update.md:56]: p.s. can you `update --to-remote` data `import-url`ed without `--to-remote`? I assume there's no restriction to do so but just curious.

Comment (isidentical) [reply content/docs/command-reference/update.md:56]: > p.s. can you update --to-remote data import-urled without --to-remote

yes

**Pull request merged**